### PR TITLE
Bug: Add validations to Expand cluster input field

### PR DESCRIPTION
**Pull request by a2batic**
Fixes: https://jira.coreos.com/browse/RHSTOR-748
Signed-off-by: Kanika <kmurarka@redhat.com>

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx [hunk 1]
```diff
@@ -29,8 +30,10 @@ export const AddCapacityModal = withHandlePromise((props: AddCapacityModalProps)
   const submit = (event: React.FormEvent<EventTarget>) => {
     event.preventDefault();
     setProgress(true);
-    const presentCapacity = _.get(ocsConfig, 'spec.storageDeviceSets[0].count');
-    const newValue = parseInt(presentCapacity, 10) + parseInt(requestSizeValue, 10) * 3;
+    const presentCount = _.get(ocsConfig, 'spec.storageDeviceSets[0].count');
+    const negativeValue = Number(requestSizeValue) < 0 ? -1 : 1;
+    const newValue =
+      (parseInt(presentCount, 10) + Math.abs(Number(requestSizeValue)) * 3) * negativeValue;
```
Comment (gnehapk) [inline frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx]: Any reason for using different functions `parseInt` and `Number` for converting to number in a single line. I feel we should stick with a single method only if it fulfills the requirement.

Comment (a2batic) [reply frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx]: ack, will use `Number` in both places

**Pull request merged**